src/Components/Ratings/Ratings.jsx

```diff
@@ -0,0 +1,5 @@
+export const Ratings = (props) => {
+    return (
+        <p className="flex-row align-center rating-box gap-8-px"><span className="rating-value">{props.text}</span><img className="star-img" src="https://img.icons8.com/color/48/000000/filled-star--v1.png" alt="rating"/></p>
```

Comment:
no idea why but in github it just puts your code into a single line and it reduces the readability, if you've typed it the same way in your editor it would be amazing if you could give line breaks.

Reply:
No, actually I structured it better way, don't know why this is happening